bucket/opera.json

```diff
@@ -35,17 +35,23 @@
     ],
     "persist": "profile",
     "checkver": {
-        "url": "http://get.opera.com/ftp/pub/opera/desktop/",
-        "regex": "([\\d.]+)/",
-        "reverse": true
+        "script": [
```

Comment:
Why do we need a script when a simple reversed regex result works fine? Did I missed something?

Reply:
Regex returns first match on the page, but versions are not properly sorted on the FTP page.